src/bsconfig.rs

```diff
@@ -239,7 +245,17 @@ pub fn read(path: String) -> Config {
 }
 
 fn check_if_rescript11_or_higher(version: &str) -> bool {
-    version.split('.').next().unwrap().parse::<usize>().unwrap() >= 11
+    // Non stable rescript versions might contain non-numeric characters
+    let filter_non_number_chars = |s: &str| s.chars().filter(|c| c.is_numeric()).collect::<String>();
+
+    version
+        .split('.')
+        .next()
+        .map(filter_non_number_chars)
+        .unwrap()
+        .parse::<usize>()
+        .unwrap()
+        >= 11
```

Comment:
I'm not sure this cuts it - I think you can do wildcards like: `rescript: *`, which I think would fail to parse, and that would throw in this case. To really check this, we should probably parse the output of the `rescript -v`. But, I'm ok to use this for the time being, albeit with some small changes:

- Return `Result<bool>` here (probably my preference)
- Default to false
- Or write some heavy tests around this

Reply:
I see we also hacked this in at some point. But perhaps, while we're touching it, maybe we just fix it up nicely 👌 - If I have some time today, I'll try and conjure up a version as a suggestion

Reply:
I will go with the `Result`, then we can log an error at the call site and decide how to proceed.
In this case without uncurried mode.